Readme.md

```diff
@@ -191,6 +192,57 @@ while (keys.length) {
 
 [crockfordconvention]: http://javascript.crockford.com/code.html
 
+## Declare Classes
+
+Declare a class using the `@constructor` jsdoc annotation. Like,
```

Comment:
I am also iffy on this. The downsides I see - more overhead for developer. more weight in the codebase without more needed info. I tend to find Java-style documentation pretty hard to get through - to me, it feels like something that could be said in a single sentence and get the point across fast is belabored and made unnecessarily long.